x-pack/plugins/security_solution/public/cases/components/edit_connector/index.tsx

```diff
@@ -61,6 +63,11 @@ export const EditConnector = React.memo(
       [selectedConnector]
     );
 
+    const onError = useCallback(() => {
+      form.setFieldValue('connector', selectedConnector);
```

Comment:
there was just a [refactor that destructures](https://github.com/elastic/kibana/pull/71237) everything from `form`, so i think we should follow that convention